### PR TITLE
Fix for pvr issues on p9, p10

### PR DESCRIPTION
**Pull request by kalshett**
Fixed p9 dd2.1 pvr value and p10 revision issue

Signed-off-by: Kalpana Shetty <kalshett@in.ibm.com>

### CODE REVIEW

**Comment by kalshett** [comment]
### P10 runs ###
[root@ltcden14-lp10 cpu]# avocado run pvr.py
JOB ID     : c60a768a31e6d4254d097c810717867607e38209
JOB LOG    : /home/kal/avocado-fvt-wrapper/results/job-2021-06-24T06.20-c60a768/job.log
 (1/1) pvr.py:pvr.test: PASS (1.24 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/kal/avocado-fvt-wrapper/results/job-2021-06-24T06.20-c60a768/results.html
JOB TIME   : 1.98 s

[job_p10.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/6708405/job_p10.log)

#### P9 runs ###
[root@f200-lp2 cpu]# avocado run pvr.py
JOB ID     : 25d6b3382ddd6351253fe743f01d154ced717e5f
JOB LOG    : /home/kal/avocado-fvt-wrapper/results/job-2021-06-24T06.21-25d6b33/job.log
 (1/1) pvr.py:pvr.test: PASS (0.99 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/kal/avocado-fvt-wrapper/results/job-2021-06-24T06.21-25d6b33/results.html
JOB TIME   : 1.57 s


[job_p9.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/6708409/job_p9.log)

**Review by nasastry — APPROVED** [review]
LGTM

File: cpu/pvr.py [hunk 1]
```diff
@@ -49,8 +49,17 @@ def setUp(self):
             self.cancel("%s package is needed for the test to be run" % pkg)
 
         val = genio.read_file("/proc/cpuinfo")
+
```
Comment (PraveenPenguin) [inline cpu/pvr.py:52]: please remove this white space

File: cpu/pvr.py [hunk 2]
```diff
@@ -49,8 +49,17 @@ def setUp(self):
...
+        for line in val.splitlines():
+            if 'revision' in line:
+                rev = (line.split('revision')[1]).split()
+                self.log.info("Revision: %s %s" % (rev, rev[1]))
+                break
+
```
Comment (PraveenPenguin) [inline cpu/pvr.py:58]: this as well

**Review by PraveenPenguin — COMMENTED** [review]
@kalshett  over all patch looks good have some minor style changes requested , please address those so we are good to go in

File: cpu/pvr.py (hunk 1 above)
Comment (kalshett) [reply cpu/pvr.py:52]: done

File: cpu/pvr.py (hunk 2 above)
Comment (kalshett) [reply cpu/pvr.py:58]: done.

**Comment by kalshett** [comment]
@PraveenPenguin resent the PR, please merge.

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**